__tests__/RelatedProducts.test.js

```diff
@@ -4,6 +4,59 @@ import App from '../client/src/components/App.jsx';
 import RelatedProducts from '../client/src/components/RelatedProducts.jsx';
 
 describe('<RelatedProducts />', () => {
+
+  const products = [{
+    id: 10,
+    name: 'test product',
+    price: '10.00',
+    imageUrls: ['fakeURL']
+  }];
+
+  const moreProducts = [
+    {
+      id: 10,
+      name: 'test product',
+      price: '10.00',
+      imageUrls: ['fakeURL']
+    },
+    {
+      id: 11,
+      name: 'test product 2',
+      price: '25.00',
+      imageUrls: ['fakeURL2']
+    },
+    {
+      id: 12,
+      name: 'test product 3',
+      price: '50.00',
+      imageUrls: ['fakeURL3']
+    },
+    {
+      id: 13,
+      name: 'test product 4',
+      price: '100.00',
+      imageUrls: ['fakeURL4']
+    },
+    {
+      id: 14,
+      name: 'test product 5',
+      price: '125.00',
+      imageUrls: ['fakeURL5']
+    },
+    {
+      id: 15,
+      name: 'test product 6',
+      price: '150.00',
+      imageUrls: ['fakeURL6']
+    },
+    {
+      id: 16,
+      name: 'test product 7',
+      price: '175.00',
+      imageUrls: ['fakeURL7']
+    },
+];
+
```

Comment:
Says ImageUrls, but your only passing an array of one url. Is this on purpose?

Reply:
It is a standard to have arrays named using a plural word. My component only renders one image (and there are multiple images for each product). So yes this was on purpose.